integration/spark/src/test/scala/org/apache/carbondata/integration/spark/testsuite/dataload/TestLoadDataGeneral.scala

```diff
@@ -154,13 +154,18 @@ class TestLoadDataGeneral extends QueryTest with BeforeAndAfterEach {
     sql("CREATE TABLE load32000chardata(dim1 String, dim2 String, mes1 int) STORED AS carbondata")
     sql("CREATE TABLE load32000chardata_dup(dim1 String, dim2 String, mes1 int) STORED AS carbondata")
     sql(s"LOAD DATA LOCAL INPATH '$testdata' into table load32000chardata OPTIONS('FILEHEADER'='dim1,dim2,mes1')")
-    intercept[Exception] {
-      sql("insert into load32000chardata_dup select dim1,concat(load32000chardata.dim2,'aaaa'),mes1 from load32000chardata").show()
-    }
+    checkAnswer(sql("select count(*) from load32000chardata"), Seq(Row(3)))
```

Comment:
i think you have handled specifically for the bad record action `FORCE`. Even test cases are also for force. Please handle for all the actions of bad record and add a test case for each action.

Reply:
It was handled for all types of action. In case of "FORCE" action, I am just converting long string to null. Now I have added testcases for all those types.